dashboard/app/helpers/users_helper.rb

```diff
@@ -527,4 +527,10 @@ def lti_user_info_required?(user)
 
     Policies::Lti.lti?(current_user)
   end
+
+  def country_code(user, request)
+    return user.country_code if user.student?
+
+    user.country_code.presence || request.country.to_s.upcase
```

Comment:
if `request.country` == 'RD', we usually treat it as 'US'. Should we convert to 'RD' -> 'US' here or update `(['US'].includes(currentUser.countryCode) || !!currentUser.usStateCode)` to check for RD too? I will let you decide.